src/Stripe.net/Stripe.net.csproj

```diff
@@ -38,6 +39,12 @@
     <CodeAnalysisRuleSet>..\_stylecop\StyleCopRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.5.0" />
+    <PackageReference Include="System.Linq.Async" Version="4.0.0" />
```

Comment:
This adds support for async enumerators to LINQ methods. We use this to convert the `IAsyncEnumerable` to a regular `IEnumerable` (so that the synchronous version `ListRequestAutoPaging` is just a thin wrapper over the async `ListRequestAutoPagingAsync`).